src/api/responses.rs

```diff
@@ -0,0 +1,254 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::ops::Deref;
+
+use axum::response::IntoResponse;
+use serde::{
+    Deserialize,
+    Serialize,
+};
+use serde_json::Value;
+
+use crate::types::LedgerInclusionState;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub(crate) enum ListenerResponse {
+    /// Response of GET /info
+    Info {
+        name: String,
+        version: String,
+        #[serde(rename = "isHealthy")]
+        is_healthy: bool,
+    },
+    /// Response of GET /api/<keyspace>/messages/<message_id>
+    /// and GET /api/<keyspace>/transactions/<transaction_id>/included-message
```

Comment:
Do we use keyspace or api version?
```suggestion
    /// Response of GET /api/<api_version>/messages/<message_id>
    /// and GET /api/<api_version>/transactions/<transaction_id>/included-message
```

Reply:
I think API version should be sufficient.